### PR TITLE
Add sphinx-copybutton

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/sphinx-copybutton/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 94593d9fd7318fa119ae4144fc99a91cb5cd4523adb363b6a93ee9b2ad0160ae
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (marcelotrevisani) [inline recipes/sphinx-copybutton/meta.yaml]: I think that could be ``noarch: python``

Comment (hmaarrfk) [reply recipes/sphinx-copybutton/meta.yaml]: thanks

Comment (hmaarrfk) [reply recipes/sphinx-copybutton/meta.yaml]: also python 3 only.

Comment (hmaarrfk) [reply recipes/sphinx-copybutton/meta.yaml]: but I think that is taken care of by flit

**Pull request merged**